### PR TITLE
Update for my-courses, enrollments, organizations, activities and sir…

### PR DESCRIPTION
**Pull request by ciss1995**
Update for my-courses, enrollments, organizations, activities and siren-sdk

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -3484,7 +3488,7 @@
       }
     },
     "d2l-sequences": {
-      "version": "github:brightspaceHypermediaComponents/sequences#ebb73ed14b0c362baaa24d89e778bfbf91c9672b",
+      "version": "github:brightspaceHypermediaComponents/sequences#9d400933e98800bd7fbe2ebe267cdfa69f2c1c97",
```
Comment (edwardg8) [inline package-lock.json]: i'm not sure about `d2l-sequences`, either have to let the owner know or remove it?

Comment (m6jones) [reply package-lock.json]: @jwong-d2l

Comment (ciss1995) [reply package-lock.json]: I removed it from the PR.

File: package-lock.json [hunk 2]
```diff
@@ -3484,7 +3488,7 @@
       }
     },
     "d2l-sequences": {
-      "version": "github:brightspaceHypermediaComponents/sequences#9d400933e98800bd7fbe2ebe267cdfa69f2c1c97",
+      "version": "github:brightspaceHypermediaComponents/sequences#ebb73ed14b0c362baaa24d89e778bfbf91c9672b",
       "from": "github:brightspaceHypermediaComponents/sequences",
```
Comment (edwardg8) [inline package-lock.json]: `d2l-sequences` is still here?

Comment (ciss1995) [reply package-lock.json]: Yeah, but if you look at it, it is the reverse of the first commit lol.

**Pull request merged**